src/CMakeLists.txt

```diff
@@ -117,6 +117,21 @@ target_link_libraries(
   Qt5::Network
   Qt5::Widgets)
 
+set(USE_OPENSSL FALSE)
```

Comment:
As discussed in #1022 , we do not want this implementation.

Reply:
I just take this code from the upstream last Thursday.
Of cause it can be replaced with the latest version.